spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py

```diff
@@ -0,0 +1,330 @@
+from spikeextractors import RecordingExtractor,SortingExtractor
+from spikeextractors.extractors.bindatrecordingextractor import BinDatRecordingExtractor
+import numpy as np
+from pathlib import Path
+from spikeextractors.extraction_tools import check_get_traces_args,check_valid_unit_id,read_binary
+from bs4 import BeautifulSoup
+import os
+
+class NeuroscopeRecordingExtractor(RecordingExtractor):
```

Comment:
I think we can change that and inherit from the BinDatRecordingExtractor later

Reply:
Copying the structure from the phy version worked well, passing tests on my end.